src/main/java/hudson/plugins/emailext/ExtendedEmailPublisher.java

```diff
@@ -456,6 +456,7 @@ private boolean executePresendScript(ExtendedEmailPublisherContext context, Mime
 
             Binding binding = new Binding();
             binding.setVariable("build", context.getBuild());
+            binding.setVariable("run", context.getRun());
```

Comment:
Is it really needed? The term “build” is widely accepted, including for a `WorkflowRun`. Anyway there is no need for a `presendScript` for Workflow.